app/src/sharedTest/java/org/oppia/app/utility/EspressoTestsMatchers.kt

```diff
@@ -2,11 +2,15 @@ package org.oppia.app.utility
 
 import android.view.View
 import org.hamcrest.Matcher
+import org.oppia.app.utility.TabMatcher.Companion.matchCurrentTabTitleCheck
 
 // https://medium.com/@dbottillo/android-ui-test-espresso-matcher-for-imageview-1a28c832626f#.4snjg8frw
 /** This object mainly facilitates as bridge between test-cases and various custom test-matchers. */
 object EspressoTestsMatchers {
 
+  fun matchCurrentTabTitle(tabTitle: String): Matcher<View> {
+    return matchCurrentTabTitleCheck(tabTitle)
+  }
```

Comment:
Nit: blank newline after this method for spacing.

Reply:
Resolved